spec/controllers/authors_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe AuthorsController do
+  let(:author) { Author.create!(name: "H. Jeff", email: "jeff@sbahj.info", phone_number: "5553059028") }
+
+  describe "showing an author" do
+    it "shows an author" do
+      get :show, id: author.id
+      expect(assigns(:author)).to eq(author)
```

Comment:
regarding what @loganhasson said about `assigns` https://github.com/learn-co-curriculum/validations-in-controller-actions-rails-lab/pull/1#discussion_r47662102 i think the same applies here.